### ISSUE TITLE
DMR EP decryption 

### ISSUE THREAD

**Issue by Dianlp471** [szechyjs/dsd#101]
Please look into adding DMR EP decryption with a known key

**Comment by mattimac** [comment]
Just to clarify. Are we talking about DMRA 40-bit ARC4 encryption?

**Comment by Dianlp471** [comment]
Yes that is correct. Tait,Motorola, Hytera all use 40-bit ARC4 encryption.

**Comment by iScottybotty** [comment]
Any progress on this?

**Comment by dmaltsiniotis** [comment]
This will probably never happen per the policy outlined in the Wiki regarding encryption: https://github.com/szechyjs/dsd/wiki/Operation#encryption:
> Decryption of speech is NOT supported, even if you lawfully posses the encryption keys. Decryption support will not be added in the future as the authors wish to steer as far away from the legal issues associated with encryption as possible.
